apps/web/src/routes/NonAuthLayoutRouter/NonAuthLayoutRouter.js

```diff
@@ -62,23 +62,35 @@ export default function NonAuthLayoutRouter (props) {
     }
   }, [dispatch, setReturnToPath, returnToPath])
 
+  const getPageTitle = () => {
+    const path = location.pathname
+    if (path.includes('/signup')) return t('Sign Up for Hylo')
+    if (path.includes('/login')) return t('Sign in to Hylo')
+    if (path.includes('/reset-password')) return t('Reset Password')
+    if (path.includes('/notifications')) return t('Manage Notifications')
+    if (path.includes('/oauth/login')) return t('OAuth Login')
+    if (path.includes('/oauth/consent')) return t('OAuth Consent')
+    if (path.includes('/groups/') || path.includes('/h/use-invitation')) return t('Join Group')
+    return 'Hylo'
+  }
+
   return (
-    <Div100vh className={classes.nonAuthContainer}>
+    <Div100vh className='w-full h-full'>
       <Helmet>
-        <title>Hylo</title>
+        <title>{getPageTitle()}</title>
```

Comment:
this is not the best way to do this, each page can have its own Helmet so that the page itself is where the title is defined

Reply:
OK. So you're saying in each route I should set a <Helmet> with the appropriate title?

Reply:
set a HELMET

Reply:
yes, in the component for each route